great_expectations/checkpoint/actions.py

```diff
@@ -905,33 +903,6 @@ def _run(  # type: ignore[override] # signature does not match parent  # noqa: P
             return output
 
 
-def store_validation_results(  # noqa: PLR0913
-    validation_result_store: ValidationsStore,
-    suite_validation_result: ExpectationSuiteValidationResult,
-    suite_validation_result_identifier: ValidationResultIdentifier | GXCloudIdentifier,
-    expectation_suite_identifier: Optional[ExpectationSuiteIdentifier | GXCloudIdentifier] = None,
-    checkpoint_identifier: Optional[GXCloudIdentifier] = None,
-    cloud_mode: bool = False,
-) -> bool | GXCloudResourceRef:
```

Comment:
Made this a function of the validations store; will remove this action here entirely soon enough